docusearch/management/commands/import_documents.py

```diff
@@ -67,10 +73,10 @@ def create_document(document, release_slug):
 
     d.release_agency_slug = release_slug
 
+    doc_file = File(open(doc_path, 'rb'))
     filename = os.path.basename(doc_path)
-    doc_path = 'docusearch/pdfs/%s' % filename
-    d.path = doc_path
-    d.save()
+
+    d.original_file.save(filename, doc_file, save=True)
```

Comment:
Turns out this was a permissions issue. Eric tired it out with his keys and everything worked fine. Issues being taken care of here: 18F/DevOps/issues/242